src/report_utils.py

```diff
@@ -10,6 +10,6 @@ def get_active_s3_folders(repo:str, bucket_name:str):
     ).get_all_folder_names_in_repo_folder()
 
     folders = sorted(list(set(all_folders).intersection(set(all_branches))))
-    folders.remove(default_branch)
+    if default_branch in folders: folders.remove(default_branch)
```

Comment:
Not exactly sure how but this was throwing an error because default_branch wasn't in the list. Might want to actually figure out why rather than this junky fix

Reply:
seems it's because `folders` was empty because EDDE's S3 folder is not the same name as the repo
```python
repo = "db-equitable-development-tool"
s3_folder = "db-eddt"
```

I'll save suggested changes for the next PR